src/PostgREST/DbStructure.hs

```diff
@@ -161,22 +162,20 @@ decodeProcs =
           (body, def) = breakOn " DEFAULT " argNoMode
           (name, typ) = breakOn " " body in
       if T.null typ
-         then Nothing
+         then Just $
+           PgArg mempty (strip typ) (T.null def) isVariadic
```

Comment:
Ah.. I see what you did here. A comment would be nice - the logic about the unnamed arg is not obvious by just looking at the code.

Reply:
Hm.... could you explain to me what I did there? It's been a while :D.

So this is the `T.null typ` branch, that means the argument is unnamed and the type is actually in `name` per the `breakOn` above. That means it actually has to be `PgArg mempty (strip name) ...`, right?

Reply:
Oh, the only thing I noticed here was the lack of a name in the PgArg, so unnamed arg. I was actually hoping you clarify the other parts with the comment :D.

Reply:
I do think this logic can be improved, but for now a simple comment would help:
```suggestion
-- Handle unnamed args. TODO: refactor to types
         then Just $
           PgArg mempty (strip typ) (T.null def) isVariadic
```